cumulusci/tasks/salesforce/ActivateFlowProcesses.py

```diff
@@ -0,0 +1,53 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+
+
+class ActivateFlowProcesses(BaseSalesforceApiTask):
+    """
+    Activate the Flows with the supplied Developer Names
+    """
+
+    task_options = {
+        "developer_names": {
+            "description": (
+                "Activates Flows identified by a given list of Developer Names"
+            ),
+            "required": True,
+        },
+    }
+
+    def _init_options(self, kwargs):
+        self.options = {}
+        self.options["developer_names"] = process_list_arg(
+            self.options.get(
+                "developer_names",
+                ["Auto_Populate_Date_And_Name_On_Program_Engagement", "ape"],
+                # **kwargs,
+            )
+        )
+
+    def _run_task(self):
+        self.logger.info(
+            f"Activating the following Flows: {self.options['developer_names']}"
+        )
+        self.logger.info("Querying flow definitions...")
+        res = self.tooling.query(
+            "SELECT Id, ActiveVersion.VersionNumber, LatestVersion.VersionNumber, DeveloperName FROM FlowDefinition WHERE DeveloperName IN ({0})".format(
+                ",".join([f"'{n}'" for n in self.options.get("developer_names")])
```

Comment:
```suggestion
                ",".join([f"'{n}'" for n in self.options["developer_names"]])
```

_init_options made sure it exists, so now we can assume it does. Also, `.get()` returns None if not found, which would break since it's not iterable.